src/libMessage/ZilliqaMessage.proto

```diff
@@ -662,7 +662,10 @@ message LookupSetOfflineLookups
 
 message LookupRaiseStartPoW
 {
-    // Empty
+    required uint32 msgtype      = 1; // Only LSB used
```

Comment:
Might be easier to wrap the inputs to the signature in a nested message, so that we don't have to individually serialize them for the input to signing